packages/falter-berlin-hna-list/src/libhna/hna.c

```diff
@@ -0,0 +1,169 @@
+#include "hna.h"
+#include <stdlib.h>
+
+char *read_file(const char *filename) {
+        char *buffer;
+        FILE *f = fopen(filename, "rb");
+
+        if (!f) {
+                fprintf(stderr, "failed to open %s\n", filename);
+                return NULL;
+        }
+
+        fseek(f, 0, SEEK_END);
+        long length = ftell(f);
+        fseek(f, 0, SEEK_SET);
+        buffer = malloc(length);
```

Comment:
we should use calloc() here to please valgrind a bit

Reply:
this should be resolved in the latest commit.